tools/dotnet-fssrgen/NuGet.Config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
remove that, see https://github.com/fsharp/FSharp.Compiler.Service/pull/556#issuecomment-212895651